release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -86,6 +86,68 @@ This release introduces the following updates to the *Developer* perspective of
 [id="ocp-4-16-openshift-cli_{context}"]
 === OpenShift CLI (oc)
 
+[id="ocp-4-16-oc-mirror-v2_{context}"]
+==== oc-mirror plugin v2
+
+The oc-mirror plugin for {product-title} has been enhanced with several new features and functionalities in oc-mirror plugin v2, aimed at improving the mirroring process for operator images and other {product-title} content. 
+
+The following are the key enhancements and features in oc-mirror plugin v2:
+
+.Automatic Generation of IDMS and ITMS Objects: 
+
+oc-mirror plugin v2 automatically generates a comprehensive list of `ImageDigestMirrorSet` (IDMS) and `ImageTagMirrorSet` (ITMS) objects after each run. These objects replace the `ImageContentSourcePolicy` (ICSP) used in oc-mirror plugin v1. This enhancement eliminates the need for manual merging and cleanup of operator images and ensures all necessary images are included.
+
+.CatalogSource Objects Creation: 
+
+The plugin now generates `CatalogSource` objects for all relevant catalog indexes. This feature simplifies the application of oc-mirror's output artifacts to disconnected clusters, enhancing consistency and saving time.
+
+.Improved Verification: 
+    
+oc-mirror plugin v2 verifies that the complete image set specified in the image set config is mirrored to the registry, regardless of whether the images were previously mirrored or not. This ensures comprehensive and reliable mirroring.
+
+.Cache System: 
+    
+The new cache system replaces metadata, maintaining minimal archive sizes by incorporating only new images into the archive. This optimizes storage and improves performance.
+
+.Selective Mirroring by Date: 
+
+Users can now generate mirroring archives based on the mirroring date, allowing for the selective inclusion of new images.
+
+.Enhanced Image Deletion Control: 
+
+The introduction of a `Delete` feature replaces automatic pruning, providing users with greater control over image deletion.
+
+.Support for `registries.conf`: 
+
+oc-mirror plugin v2 supports `registries.conf`, facilitating mirroring to multiple enclaves using the same cache. This enhances flexibility and efficiency in managing mirrored images.
+
+.Operator Version Filtering: 
+
+Users can filter Operator versions by bundle name, offering more precise control over the versions included in the mirroring process.
+
+.Differences Between oc-mirror v1 and v2
+
+While oc-mirror plugin v2 brings numerous enhancements, some features from oc-mirror plugin v1 are not yet supported:
```

Comment:
Are these dev preview or tech preview? I think a support level needs to be stated.

Reply:
v2 is tech preview. Let me add it in the heading.